pkg/db/db-connection.go

```diff
@@ -0,0 +1,13 @@
+package db
+
+import (
+	"context"
+	"os"
+
+	"github.com/jackc/pgx/v4"
+)
+
+func ConnectToDB(ctx context.Context) (*pgx.Conn, error) {
+	var psqlconn = os.Getenv("POSTGRES_URI")
+	return pgx.Connect(ctx, psqlconn)
+}
```

Comment:
I think we can drop this package entirely and handle connecting to the postgres database at the main.go level.